Makefile

```diff
@@ -204,25 +204,31 @@ docs: | build deps
 
 ## RLN
 
+compile_rln =
+
 # control rln code compilation
 ifeq ($(RLN), true)
-NIM_PARAMS := $(NIM_PARAMS) -d:rln
-else ifeq ($(CI), true)
-NIM_PARAMS := $(NIM_PARAMS) -d:rln
+compile_rln = true
 endif
 
 # control compilation of rln tests that require on chain interaction
 ifeq ($(ONCHAIN_RLN), true) 
```

Comment:
Why are we still differentiating in the makefile between on-chain and off-chain RLN? Aren't we running all tests always?

Reply:
Addressed in [055b387](https://github.com/waku-org/nwaku/pull/1410/commits/055b387a3d35467ad85fe50f7576a9af1683e2b9)